crates/astria-core/src/protocol/transaction/v1/action/group/mod.rs

```diff
@@ -58,7 +58,8 @@ impl Group {
         matches!(self, Group::BundleableGeneral | Group::BundleableSudo)
     }
 
-    pub(crate) fn is_bundleable_sudo(self) -> bool {
+    #[must_use]
+    pub fn is_bundleable_sudo(self) -> bool {
```

Comment:
I guess this is now being used in sequencer somewhere? Can you please add a small doc comment?

Reply:
Done in [2270e90](https://github.com/astriaorg/astria/pull/2142/commits/2270e90cc6f02a75e4ce0ea377e084f7031cace3).